Kickstarter-iOS/Views/Controllers/EmailVerificationViewController.swift

```diff
@@ -143,9 +143,10 @@ final class EmailVerificationViewController: UIViewController, MessageBannerView
         self.delegate?.emailVerificationViewControllerDidComplete(self)
       }
 
-    self.viewModel.outputs.showSuccessBannerWithMessage
+    self.viewModel.outputs.showSuccessBannerWithMessageAndShouldShow
       .observeForUI()
-      .observeValues { [weak self] message in
+      .observeValues { [weak self] message, shouldShow in
```

Comment:
Not sure how strict we are about Swift syntax and the bool variable prefix `is` but this is readable and I don't mind keeping it as is.

Reply:
Hmm yeah good point - _should_ is probably not great, although _is_ makes more sense for properties like `isEnabled`. Maybe this should just be `showBanner`?

Reply:
Yep good call. I think `showBanner` is a more descriptive.